numba/tests/test_builtins.py

```diff
@@ -323,18 +323,6 @@ def test_bool_nonnumber_npm(self):
         with self.assertTypingError():
             self.test_bool_nonnumber(flags=no_pyobj_flags)
 
-    def test_chr(self, flags=enable_pyobj_flags):
```

Comment:
Removing this removes all references to the `chr_usecase` defined above. It may well have to be removed also.

Reply:
I also found a `cmp_usecase` that was dead, removed that too.

Reply:
top!